orchagent/portsorch.cpp

```diff
@@ -642,12 +648,23 @@ PortsOrch::PortsOrch(DBConnector *db, DBConnector *stateDb, vector<table_name_wi
 
         string portRateLuaScript = swss::loadLuaScript(portRatePluginName);
         portRateSha = swss::loadRedisScript(m_counter_db.get(), portRateLuaScript);
+
+        string nvdaPortTrimLuaScript = swss::loadLuaScript(nvdaPortTrimPluginName);
+        nvdaPortTrimSha = swss::loadRedisScript(m_counter_db.get(), nvdaPortTrimLuaScript);
     }
     catch (const runtime_error &e)
     {
         SWSS_LOG_ERROR("Port flex counter groups were not set successfully: %s", e.what());
     }
 
+    std::string portStatPlugins = portRateSha;
+
+    // Nvidia custom trim stat calculation
+    if (gMyPlatform == "x86_64-nvidia_sn5640-r0")
```

Comment:
Could you use SAI attribute capability  to check instead of platform and use MLNX_PLATFORM_SUBSTRING to use LUA plugin

Reply:
@kperumalbfn resolved